src/app.ts

```diff
@@ -1100,13 +1102,69 @@ class App {
     router.get("/manualjournals", async (req: Request, res: Response) => {
       try {
         //GET ALL
-        const apiResponse = await xero.accountingApi.getManualJournals(req.session.activeTenant.tenantId);
+        const getManualJournalsResponse = await xero.accountingApi.getManualJournals(req.session.activeTenant.tenantId);
+
         // CREATE
+        const manualJournals: ManualJournals = {
+          manualJournals: [
+            {
+              date: "2020-03-13",
+              status: ManualJournal.StatusEnum.DRAFT,
+              narration: "Accrued expenses - prepaid insurance adjustment for March 2020",
+              journalLines: [
+                {
+                  lineAmount: 55.00,
+                  accountCode: "433"
+                },
+                {
+                  lineAmount: -55.00,
+                  accountCode: "620"
+                }
+              ]
+            }
+          ]
+        };
+
+        const createManualJournalResponse = await xero.accountingApi.createManualJournals(req.session.activeTenant.tenantId, manualJournals);
+
+        // CREATE MANUAL JOARNAL ATTACHMENT BY FILENAME
+        const fileName = "xero-dev.png";  // {String} The name of the file being attached to a ManualJournal 
+        const path = require("path");
+        const mime = require("mime-types");
+        const pathToUpload = path.resolve(__dirname, "../public/images/xero-dev.png"); // determine the path to your file
+
+        // You'll need to add the import below to read your file
+        // import * as fs from "fs";
+        const body = fs.createReadStream(pathToUpload); // {fs.ReadStream} read the file
+        const contentType = mime.lookup(fileName);
+        const createManualJournalAttachmentByFileNameResponse: any = await xero.accountingApi.createManualJournalAttachmentByFileName(req.session.activeTenant.tenantId, createManualJournalResponse.body.manualJournals[0].manualJournalID, fileName, body, {
```

Comment:
I think `journalId` is used multiple?

```suggestion
        const contentType = mime.lookup(fileName);
        const journalId = createManualJournalResponse.body.manualJournals[0].manualJournalID
        const createManualJournalAttachmentByFileNameResponse: any = await xero.accountingApi.createManualJournalAttachmentByFileName(req.session.activeTenant.tenantId, journalId, fileName, body, {
```